app/controllers/candidate_interface/decoupled_references/request_controller.rb

```diff
@@ -4,8 +4,14 @@ class RequestController < BaseController
       before_action :set_reference
       before_action :prompt_for_candidate_name_if_not_already_given, only: :create
 
+      # TODO: Replace this action when integrating with ref summary page
+      def start
+        @request_form = Reference::RequestForm.build_from_reference(@reference)
+      end
```

Comment:
this is a temporary action that we can get rid of once this work is integrated properly